Introduction.md

```diff
@@ -44,13 +44,13 @@ It should usually be broadcast together with the corresponding CET.
 
 ### Contract Execution Transaction (CET)
 
-A transaction spending from the [fund transaction](#fund-transaction) whose outputs represent a possible outcome of a DLC contract.
+A transaction spending from the [funding transaction](#funding-transaction) whose outputs represent a possible outcome of a DLC contract.
 The first output can be spent either by one of the contracting party with knowledge of an oracle signature over the outcome, or by their counter-party after a given timeout.
 The second output can be spent directly.
 
 ### Funding transaction
 
-An irreversible on-chain transaction locking the contract collateral in an output that can only be spent by mutual consent.
```

Comment:
To be clear, all on-chain transactions are equally "irreversible", so this should at least clarified (but I think it could be dropped altogether)

Reply:
Can't reverse this transaction :P 

https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b